integration-tests/build.gradle

```diff
@@ -5,6 +5,7 @@ subprojects {
     apply plugin: 'java'
 
     ext.enclaveMode = findProperty("enclaveMode")?.toString()?.toLowerCase() ?: "simulation"
+    ext.runtimeType = findProperty("runtimeType")?.toString() ?: "GraalVM"
```

Comment:
Is this particular variable used elsewhere?

Reply:
> You'll need to update ci_integration_tests.sh as well to be able to create a separate job on TC.

Script & TC updated.

Reply:
I just copied the line above without thinking too much. Does this need changing?

Reply:
I believe you can change all the `findProperty("runtimeType")` to something like `ext.runtimeType`

Reply:
Yeah, not sure why I did this. Fixed.